.teamcity/ETL/Project.kt

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+package etl
+
+import jetbrains.buildServer.configs.kotlin.v2019_2.*
+
+object ETLProject : Project({
+    id("Buzz_ETL")
+    name = "ETL"
+    description = "Buzz ETL"
+
+    buildType(etl.buildTypes.PullRequestETLBuild)
+    buildType(etl.buildTypes.BranchETLBuild)
+    buildType(etl.buildTypes.DeployETLBuild)
+
+    params{
+        param("project.directory", "./EdFi.Buzz.Etl");
+        param("octopus.release.version","<placeholder value>")
+        param("octopus.release.project", "Buzz ETL")
+        param("octopus.project.id", "Projects-111")
```

Comment:
This project id may need to change at some point. One of us needs to get around to doing more work in Octopus Deploy. I told David I would do that a few weeks ago and I haven't been able to prioritize it yet. Still hoping to do so soon.

So no action to take here. Just calling it out as a reminder.